source/dub/dub.d

```diff
@@ -313,8 +313,11 @@ class Dub {
 			recipe_content = file_content[0 .. idx].strip();
 		} else throw new Exception("The source file must start with a recipe comment.");
 
+		auto nidx = recipe_content.indexOf('\n');
+
 		auto idx = recipe_content.indexOf(':');
-		enforce(idx > 0, "Missing recipe file name (e.g. \"dub.sdl:\") in recipe comment");
+		enforce(idx > 0 && (nidx < 0 || nidx > idx),
```

Comment:
What's the reason for `nidx < 0`?
Currently a package format isn't valid without a `name`.
Or are you silently considering merging https://github.com/dlang/dub/pull/1081? :)